MPC/config/opendds.mpb

```diff
@@ -11,3 +11,11 @@ feature(opendds) : taflib, dcps, dcps_ts_defaults {
   IDL_Files {
   }
 }
+
+feature(java) {
+  requires      += opendds
+  includes      += $(DDS_ROOT)/java/idl2jni/runtime
+  verbatim(gnuace, macros) {
```

Comment:
Make this `verbatim(gnuace, macros, 1)`, without the 1 you are replacing any other macros set